migrations/20181021111747-create-flight.js

```diff
@@ -14,8 +14,8 @@ module.exports = {
       },
     },
     flightNumber: Sequelize.STRING,
-    departureDateTime: Sequelize.BIGINT,
-    arrivalDateTime: Sequelize.BIGINT,
+    departureDateTime: Sequelize.DATE,
+    arrivalDateTime: Sequelize.DATE,
```

Comment:
We talked about storing these as timestamps, but I guess there is no particular reason and we could convert it if needed.

Reply:
We do. The big int I assume it was supposed to work for UNIX timestamp e.g. `1541930041`, but this is not good practice as you would have to do computations everytime you save/fetch a date to convert it from or to UNIX timestamp. Instead, see the screenshot below, it specifies in the comment the format in postgres 🙌
<img width="1105" alt="screenshot 2018-11-11 at 10 52 54" src="https://user-images.githubusercontent.com/1295800/48311427-feab4c80-e59f-11e8-8371-5ac0139ee3ae.png">